en/install.md

```diff
@@ -121,7 +121,20 @@ needs to with:
 If you require traffic between your Snap Store Proxy and the internet to go via
 another HTTP proxy, you can configure your Snap Store Proxy to do so with:
 
-    sudo snap-proxy config proxy.https.proxy=myproxy.internal:3128
+    sudo snap-proxy config proxy.https.proxy="https://myproxy.internal:3128"
+
+### Custom CA Certificates
+
+If you require your Snap Store Proxy to talk to the internet via an HTTPS proxy
+that performs deep packet inspection (i.e. a MITM HTTPS proxy), then you can
```

Comment:
I'm wondering about the language here - it's not specifically the DPI part, it's if it's using a CA that the proxy doesn't know about (e.g. an internal one) - thoughts?

Reply:
Very good point, thanks! Fixed now I think.